README.md

```diff
@@ -4,28 +4,39 @@ ONTOCHAIN Demo
 </h1>
 <br>
 
+
 ## Summary
 
-This demo shows an authentication flow using the [DIDSIOP library](#did-auth-siop). The underlying technology allows a
-user to authenticate to a provider by scanning a QR code that is presented on by the relying party (RP). For this demo
-project, the [Demo WEB](#onto-web-demo) app is the relying party (RP) and the [Demo RN](#demo-rn) is the the OpenID
-provider (OP). The flow starts with the RP where the user clicks a login button on the web page, after which a QR code
-is generated. The user scans the QR code using the Demo RN app, which in turn requests access to the protected resources
-hosted by the RP. Since the user is not logged in, an authentication request will be sent back to the OP. If the request
-is valid, the OP sends an authentication response back to the RP, which verifies the validity of response. If all steps
-succeed, access to the protected resources is granted to the user. The user will now be redirected on the web app to
-view the protected resource. [GimlyID-QR-Code](#gimlyid-qr-code) and [GimlyID-QR-Code-Scanner](#gimlyid-qr-code-scanner)
-are employed to respectively generate and read the QR code used in the authentication process.
+This demo shows an authentication flow using the [DIDSIOP library](#did-auth-siop). The underlying technology allows a user to authenticate to a provider by scanning a QR code that is presented on by the relying party (RP). For this demo project, the [Demo WEB](#onto-web-demo) app is the relying party (RP) and the [Demo RN](#demo-rn) is the OpenID provider (OP). The flow starts with the RP where the user clicks a login button on the web page, after which a QR code is generated. The user scans the QR code using the Demo RN app, which in turn requests access to the protected resources hosted by the RP. Since the user is not logged in, an authentication request will be sent back to the OP. If the request is valid, the OP sends an authentication response back to the RP, which verifies the validity of response. If all steps succeed, access to the protected resources is granted to the user. The user will now be redirected on the web app to view the protected resource. [GimlyID-QR-Code](#gimlyid-qr-code) and [GimlyID-QR-Code-Scanner](#gimlyid-qr-code-scanner) are employed to respectively generate and read the QR code used in the authentication process.
+
 
 ## Authentication Flow
+
 ![ONTO Demo Flow](https://lucid.app/publicSegments/view/02cb0788-5fe6-4fcd-91f6-3f82d6ca1402/image.png)
 
+
 ## ONTO web demo
+
 This is the web application (RP) that generates the QR code and starts the authentication process;
+
 ###Setup
+
+#### Build & start
+
+clone the repository:
+
+```bash
+cd <workspace>
+git clone git@github.com:Sphereon/onto-web-demo.git
```

Comment:
Could not find this repo (git@github.com:Sphereon/onto-web-demo.git). 
Should it be  ssh: git@github.com:Gimly-Blockchain/did-siop-auth-web-demo.git
or https: https://github.com/Gimly-Blockchain/did-siop-auth-web-demo.git ?